docs/docs/versions/v0_3/index.mdx

```diff
@@ -205,3 +207,48 @@ class FooParser(BaseOutputParser):
 
 FooParser.model_rebuild()
 ```
+
+
+## Migrate using langchain-cli
```

Comment:
should this be in the "How to update your code" section?

Reply:
I think let's reference, but not move since it's just one part of how to update yoru code and doesn't handle most of the work associated with the pydantic migration

Reply:
maybe worth making that explicit here then? that this doesn't handle pydantic migration